src/features/user/Account/components/AccountRecord.tsx

```diff
@@ -305,7 +305,18 @@ export const showStatusNote = (record: PaymentHistoryRecord): ReactElement => {
       return <p className={styles.donationNote}>{showDonationNote()}</p>;
     case 'in-dispute':
       return (
-        <p className={styles.donationNote}>{t('donationNote.in-dispute')}</p>
+        <p className={styles.donationNote}>
+          {t.rich('donationNote.in-dispute', {
+            supportLink: (chunk) => (
+              <a
+                className={styles.supportLink}
```

Comment:
Suggest using the `planet-links` class from `global.scss` here. It is meant for use while showing links across the app.